add restore from s3/oss (#22208)
pingcap/docs: tidb-cloud/premium/backup-and-restore-premium.md

3. On the **Restore** page, follow the same steps as [Restore to a new instance](#restore-to-a-new-instance) to restore the backup to a new instance.

> 

Reviewer: ```suggestion
```

- **Credentials**: You have the access key and secret key with sufficient permissions to access the backup files.

#### Steps

1. Log in to the [TiDB Cloud console](https://tidbcloud.com), and then navigate to the [**TiDB Instances**](https://tidbcloud.com/tidbs) page. In the upper-right corner, click **...** , and then click **Restore from Cloud storage**.

2. On the **Select Backup Storage Location** page, provide the following information, and then click **Verify Backup and Next**:
    - **Cloud Provider**: select the provider where your backup file is located
    - **Backup Files URI**: enter the URI of the top-level folder that contains your backup files
    - **Authentication**: enter your access key and secret key

3. If the verification is successful, the **Restore to a New Instance** page appears. Review the backup metadata information displayed at the top of the page:
    - If the information is incorrect, click **Previous** at the bottom of the page to return to the storage configuration.
    - If the information is correct, follow the same steps as [Restore to a new instance](#restore-to-a-new-instance) to configure and create your new instance.

> **Note:**
>

> > If the region of the new instance differs from the region of the storage bucket, additional **cross-region data transfer fees** may apply.

Reviewer: ```suggestion
> If the instance and the storage bucket are located in different regions, additional cross-region data transfer fees might apply.
```
